python/graphscope/interactive/query.py

```diff
@@ -141,6 +147,27 @@ def execute(self, query):
             raise RuntimeError("Interactive query is closed.")
         return self._client.submit(query)
 
+    def traversal_source(self):
+        """Create a GraphTraversalSource and return.
+        Once "g" has been created using a connection, we can start to write
+        Gremlin traversals to query the remote graph.
+
+        Examples:
+
+            ..code:: python
```

Comment:
Bad indent.

```
Examples:

.. code:: python

    xxxxx
```

Reply:
I think sections are supposed to indent after the section header?
ref: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

Reply:
I didn't realize the `Examples:` is a directive here, but please double check the generated doc.